### DISCUSSION TITLE
Cyrillic

### DISCUSSION THREAD

**Discussion by liyanic** [Chlumsky/msdfgen#202] [Q&A] [answered]
Somehow I cant generate Cyrillic. Is the support given ? Thanks for your help

`msdf-atlas-gen.exe -font OpenSans_Condensed-Bold.ttf -charset 'я' -type mtsdf -imageout test.png -json test.json
Neither atlas size nor glyph size selected, using default...
Failed to load character set specification.`

**Comment by Chlumsky** [comment, marked as answer]
This isn't even the right repository but yes, Cyrillic is supported just like any other part of Unicode. One problem is that `-charset` expects a filename of a text file (which should contain `'я'` in UTF-8 encoding) Also, I'm not sure if passing Unicode characters on the command line will work, so personally I would try to avoid that. You can instead use the Unicode value, in this case 0x44f, so for example:

`msdfgen -font OpenSans_Condensed-Bold.ttf 0x44f` or
`msdf-atlas-gen -font OpenSans_Condensed-Bold.ttf -chars 0x44f`
